frontend/src/scenes/dashboard/NewDashboardModal.tsx

```diff
@@ -1,21 +1,166 @@
 import { useActions, useValues } from 'kea'
+import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
+import { LemonModal } from 'lib/lemon-ui/LemonModal'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { dashboardTemplatesLogic } from 'scenes/dashboard/dashboards/templates/dashboardTemplatesLogic'
+import { DashboardTemplateVariables } from './DashboardTemplateVariables'
+import { LemonButton } from '@posthog/lemon-ui'
+import { dashboardTemplateVariablesLogic } from './DashboardTemplateVariablesLogic'
+import { DashboardTemplateType } from '~/types'
+import { useEffect, useState } from 'react'
+
 import { Field } from 'lib/forms/Field'
-import { LemonButton } from 'lib/lemon-ui/LemonButton'
 import { AvailableFeature } from '~/types'
 import { LemonSelect } from 'lib/lemon-ui/LemonSelect'
 import { PayGateMini } from 'lib/components/PayGateMini/PayGateMini'
-import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
 import { LemonInput } from 'lib/lemon-ui/LemonInput/LemonInput'
 import { LemonTextArea } from 'lib/lemon-ui/LemonTextArea/LemonTextArea'
 import { DASHBOARD_RESTRICTION_OPTIONS } from './DashboardCollaborators'
-import { LemonModal } from 'lib/lemon-ui/LemonModal'
 import { Form } from 'kea-forms'
-import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
-import { FEATURE_FLAGS } from 'lib/constants'
 import { userLogic } from 'scenes/userLogic'
-import { dashboardTemplatesLogic } from 'scenes/dashboard/dashboards/templates/dashboardTemplatesLogic'
 
-export function NewDashboardModal(): JSX.Element {
+function FallbackCoverImage({ src, alt, index }: { src: string | undefined; alt: string; index: number }): JSX.Element {
+    const [hasError, setHasError] = useState(false)
+    const [color, setColor] = useState('#ff0000')
+
+    const handleImageError = (): void => {
+        setHasError(true)
+    }
+
+    const colors = ['#ff0000', '#ffa500', '#ffff00', '#008000', '#0000ff', '#4b0082', '#ee82ee', '#800080', '#ffc0cb']
+
+    useEffect(() => {
+        setColor(colors[(index * 3) % colors.length])
```

Comment:
since that would make the index<->color stable then I think you could skip the `useEffect` too